src/screens/ProfileScreen/elements/FollowInfoRow.js

```diff
@@ -57,10 +66,10 @@ const FollowInfoRow = ({
 };
 
 let styles = StyleSheet.create({
-  following: {marginLeft: 18},
   textTotal: {
     fontSize: 14,
-    color: COLORS.signed_primary,
+    fontWeight: '600',
+    color: COLORS.black000,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The fontWeight property is missing in the old hunk, which means it's a new addition. If this change was intentional and tested across different platforms (iOS and Android), then it's fine. However, please note that not all platforms support every weight value. For example, Android only supports 'normal' (400) and 'bold' (700). Other values might work on iOS but have no effect on Android.
```diff
textTotal: {
  fontSize: 14,
- fontWeight: '600',
+ fontWeight: Platform.OS === 'ios' ? '600' : 'bold', // Use 'bold' for Android
  color: COLORS.black000,
  paddingRight: 4
},
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->